rastervision_pytorch_backend/rastervision/pytorch_backend/examples/chip_classification/spacenet_rio.py

```diff
@@ -96,8 +97,27 @@ def make_scene(scene_info):
     else:
         model = ClassificationModelConfig(backbone=Backbone.resnet50)
 
+    if external_loss:
+        external_loss_def = ExternalModuleConfig(
+            github_repo='AdeelH/pytorch-multi-class-focal-loss',
+            name='focal_loss',
+            entrypoint='focal_loss',
+            force_reload=False,
+            entrypoint_kwargs={
+                'alpha': [.75, .25],
+                'gamma': 2
+            })
+    else:
+        external_loss_def = None
+
     solver = SolverConfig(
-        lr=1e-4, num_epochs=20, test_num_epochs=3, batch_sz=32, one_cycle=True)
+        lr=1e-2,
```

Comment:
Did you mean to change the LR? I would have thought that LR would not work.

Reply:
Nope. Didn't mean to commit this.